warehouse/ingest-core/src/main/java/datawave/ingest/data/config/DataTypeOverrideHelper.java

```diff
@@ -208,4 +208,12 @@ public void closeFilters() {
             this.filters = null;
         }
     }
+
+    public List<? extends RecordFilter> getFilters() {
+        return filters;
+    }
```

Comment:
Do you want callers to be allowed to mutate the list of filters?
If not, you could return an unmodifiable copy

Reply:
I don't believe so, I'll make the update